src/main/plugins/org.dita.base/build_preprocess_template.xml

```diff
@@ -54,7 +54,15 @@ See the accompanying LICENSE file for applicable license.
     
     <basename property="dita.input.filename" file="${args.input}" />
```

Comment:
You can just modify the branch locally and then force push to your GitHub remote to update the PR.